src/packages/web/icons.gleam

```diff
@@ -1,7 +1,329 @@
-import lustre/attribute.{attribute}
+import lustre/attribute.{attribute, class}
 import lustre/element.{type Element}
 import lustre/element/svg
 
+pub fn packages() -> Element(Nil) {
+  svg.svg(
```

Comment:
This is turning into quite a bit of content. How about we have these as regular images so they can be cached? Rather than sending this content on every page load and duplicating it with every package.